pkg/resourcemanager/apis/config/validation/validation.go

```diff
@@ -26,6 +26,7 @@ import (
 
 	"github.com/gardener/gardener/pkg/logger"
 	"github.com/gardener/gardener/pkg/resourcemanager/apis/config"
+	corevalidation "github.com/gardener/gardener/pkg/utils/validation/kubernetes/core"
```

Comment:
Maybe alias it with `kubernetescorevalidation` and add this to the linter to enforce this alias